File: app/src/main/java/com/ytempest/wanandroid/dialog/LoadingDialog.java
package com.ytempest.wanandroid.dialog;

import android.content.Context;
import androidx.annotation.NonNull;

import com.ytempest.layoutinjector.annotation.InjectLayout;
import com.ytempest.wanandroid.R;
import com.ytempest.wanandroid.base.BaseDialog;

/**
 * @author heqidu
 * @since 2020/8/13
 */
@InjectLayout(R.layout.dialog_loading)
public class LoadingDialog extends BaseDialog {

    public LoadingDialog(@NonNull Context context) {
        super(context, R.style.common_dialog_transparent);
        setCanceledOnTouchOutside(false);
    }
}

File: app/src/main/java/com/ytempest/wanandroid/dialog/LoadingDialog.kt
package com.ytempest.wanandroid.dialog

import android.content.Context
import com.ytempest.wanandroid.R
import com.ytempest.wanandroid.base.BaseDialog
import com.ytempest.wanandroid.databinding.DialogLoadingBinding

/**
 * @author heqidu
 * @since 21-2-22
 */
class LoadingDialog(context: Context) : BaseDialog<DialogLoadingBinding>(context, R.style.common_dialog_transparent) {

    init {
        setCanceledOnTouchOutside(false)
    }
}